bin/server.js

```diff
@@ -4,11 +4,15 @@
 
 var restify = require('restify')
 var config = require('../config')
-var logConfig = config.get('logging')
-var log = require('../log')(logConfig.level, logConfig.app)
+
+var log = require('../log')('server')
+var mailerLog = require('../log')('mailer')
+var mailConfig = config.get('mail')
+
 var packageJson = require('../package.json')
 var P = require('bluebird')
-var Mailer = require('../mailer')(log)
+// NOTE: Mailer is also used by fxa-auth-server directly, the module has two different logging interfaces
+var Mailer = require('../mailer')(mailerLog)
```

Comment:
For readability, it may make sense to move the declaration of `mailerLog` down so that it's underneath the comment as well.